oci/defaults.go

```diff
@@ -154,6 +154,10 @@ func DefaultLinuxSpec() specs.Spec {
 			{Type: "pid"},
 			{Type: "ipc"},
 		},
+		// Always use MS_PRIVATE for rootfs propagation to make sure that we
+		// don't leak any MS_UNBINDABLE mounts into the container's namespace
+		// while its resolving volume bindmounts.
+		RootfsPropagation: "private",
```

Comment:
MS_SLAVE|MS_REC is libcontainer default. I thought somebody sets rootfsPropagation MS_PRIVATE | MS_REC in normal cases so libcontainer default does not kick in. Can't find though who does it.

Thought MS_SLAVE | MS_REC makes sense in a way that if anything leaks into container from host, it has slave relationship and if it is unmounted on host, it will get unmounted from container too, leading to less issues of busy mounts.

But, if we apply any property recursively (rshared, rprivate, rslave) on `/`, it will reset `unbindable` property of mount. So I doubt we can apply anything recursively on `/`